src/reverting/installing.md

```diff
@@ -0,0 +1,17 @@
+# Installing chromeOS
+
+::: tip
+If recovery fails, unplug the battery since recovery also attempts to update the firmware.
```

Comment:
Could also be jumper, whatever you did last time to disable wp

Reply:
do you know other people in the disc that needed to undo jumper/ccd to flash recovery?

Reply:
It's just that you need to disable hardware write protect, not all devices disable hardware write protect through battery